lib/widgets/pages/settings/your_profile_page.dart

```diff
@@ -18,80 +19,85 @@ class YourProfilePage extends StatelessWidget {
-        } else if (state is UserError) {
-          //FIXME: display error page?
-          return const SizedBox.shrink();
-        } else if (state is UserLoaded) {
-          return Column(
-            children: [
-              const Gap(24),
-              const CircleAvatar(radius: 54),
-              const Gap(12),
-              Text(
-                user.name,
-                style: AppTextStyle.sectionTitle,
-                textAlign: TextAlign.center,
-              ),
-              const Gap(8),
+    return Column(
+      children: [
+        const Gap(24),
+        const CircleAvatar(radius: 54),
+        const Gap(12),
+        Text(
+          user.name,
+          style: AppTextStyle.sectionTitle,
+          textAlign: TextAlign.center,
+        ),
+        const Gap(8),
+        //FIXME: lookup on programme id
+        Text('${user.programmeId}', style: AppTextStyle.explainer),
+        const Gap(24),
+        SettingsGroup(
+          title: Strings.settingsGroupProfile,
+          description: 'These settings affect your appearance in Leaderboards.',
+          listItems: [
+            SettingListEntry(
+              name: Strings.name,
+              valueWidget: SettingDescription(text: user.name),
+              onTap: () {},
+            ),
+            SettingListEntry(
+              name: Strings.occupation,
               //FIXME: lookup on programme id
-              Text('${state.user.programmeId}', style: AppTextStyle.explainer),
-              const Gap(24),
-              SettingsGroup(
-                title: Strings.settingsGroupProfile,
-                listItems: [
-                  SettingListEntry(
-                    name: Strings.name,
-                    valueWidget: SettingDescription(text: user.name),
-                    onTap: () {},
-                  ),
-                  SettingListEntry(
-                    name: Strings.occupation,
-                    //FIXME: lookup on programme id
-                    valueWidget:
-                        SettingDescription(text: '${state.user.programmeId}'),
-                    onTap: () {},
-                  ),
-                  SettingListEntry(
-                    name: Strings.changeProfilePicture,
-                    valueWidget: const SettingDescription(),
-                    onTap: () {},
-                  ),
-                ],
+              valueWidget: SettingDescription(text: '${user.programmeId}'),
+              onTap: () {},
+            ),
+            SettingListEntry(
+              name: Strings.changeProfilePicture,
+              valueWidget: const SettingDescription(),
+              onTap: () {},
+            ),
+            SettingListEntry(
+              name: deviceIsSmall(context)
+                  ? Strings.appearAnonymousSmall
+                  : Strings.appearAnonymous,
+              valueWidget: Switch(
+                value: user.privacyActivated,
+                onChanged: (privacyActived) async {
```

Comment:
can you verify this value is the expected one? as in true when toggled on and false when off

Reply:
looks correct to me